integration_tests/src/divviup_api_client.rs

```diff
@@ -21,7 +21,7 @@ pub enum ApiVdaf {
     /// Corresponds to Prio3Count
     Count,
     Histogram {
-        buckets: Vec<u64>,
+        buckets: usize,
```

Comment:
If we rename the parameter to `length` in the aggregator API, we'll want to do the same in the control plane API.

Reply:
https://github.com/divviup/divviup-api/issues/410 for the divviup-api side of this.